ingest/src/main/java/com/connexta/ingest/config/ApplicationConfiguration.java

```diff
@@ -19,6 +20,6 @@
 public class ApplicationConfiguration {
   @Bean
   public RestTemplate restTemplate(RestTemplateBuilder builder) {
-    return builder.build();
+    return builder.errorHandler(new DefaultResponseErrorHandler()).build();
```

Comment:
Does the default handler need to be set explicitly?

Reply:
No, but I'm doing it here for clarity since we didn't realize it exists. 
Previously, I was using a custom error handler so this demonstrates how we would use one if we need to swap in the future.